packages/gunshi/src/definition.ts

```
@@ -15,17 +15,52 @@
+ * Extended command type with extension support
+ */
+export interface ExtendedCommand<
+  A extends Args = Args,
+  E extends Record<string, ContextExtension> = {}
+> extends Omit<Command<A>, 'run'> {
+  _extensions?: E
+  run?: (
+    ctx: CommandContext<A> & {
+      ext: { [K in keyof E]: ReturnType<E[K]['factory']> }
+    }
+  ) => Awaitable<void | string>
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)